weaver/wps_restapi/swagger_definitions.py

```diff
@@ -235,11 +235,13 @@ class URL(ExtendedSchemaNode):
 
 class DateTimeInterval(ExtendedSchemaNode):
     schema_type = String
-    description = "DateTime format against OGC-API - Processes,\
-    to get values before a certain date-time use '../' before the date-time,\
-    to get values after a certain date-time use '/..' after the date-time like the example,\
-    to get values between two date-times use '/' between the date-times,\
-    to get values with a specific date-time just pass the datetime."
+    description = (
+        "DateTime format against OGC-API - Processes,\
+        to get values before a certain date-time use '../' before the date-time,\
+        to get values after a certain date-time use '/..' after the date-time like the example,\
+        to get values between two date-times use '/' between the date-times,\
+        to get values with a specific date-time just pass the datetime."
```

Comment:
Still incorrect format. If you use the parenthesis, you don't need the ``\`` notation. 
You can have strings one on top of another and they are automatically concatenated.